src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -452,10 +452,13 @@ public static string TryGetAndroidJarPath (TaskLoggingHelper log, string platfor
 		{
 			var platformPath = MonoAndroidHelper.AndroidSdk.TryGetPlatformDirectoryFromApiLevel (platform, MonoAndroidHelper.SupportedVersions);
 			if (platformPath == null) {
-				if (!designTimeBuild) {
-					var expectedPath = MonoAndroidHelper.AndroidSdk.GetPlatformDirectoryFromId (platform);
-					var sdkManagerMenuPath = buildingInsideVisualStudio ? Properties.Resources.XA5207_SDK_Manager_Windows : Properties.Resources.XA5207_SDK_Manager_CLI;
-					log.LogCodedError ("XA5207", Properties.Resources.XA5207, platform, Path.Combine (expectedPath, "android.jar"), string.Format (sdkManagerMenuPath, targetFramework, androidSdkDirectory));
+				var expectedPath = Path.Combine (AndroidSdk.GetPlatformDirectoryFromId (platform), "android.jar");
+				var sdkManagerMenuPath = buildingInsideVisualStudio ? Properties.Resources.XA5207_SDK_Manager_Windows : Properties.Resources.XA5207_SDK_Manager_CLI;
+				var details = string.Format (sdkManagerMenuPath, targetFramework, androidSdkDirectory);
+				if (designTimeBuild) {
+					log.LogDebugMessage (string.Format(Properties.Resources.XA5207, platform, expectedPath, details));
```

Comment:
This makes the log message appear:

![image](https://github.com/user-attachments/assets/0f7c1eaa-0369-40b5-9239-338ed4f130d3)

Will be useful if we ever debug a design-time build in the future.